### PR TITLE
Fix typo in README.md and fix CI from forks

### PR DESCRIPTION
**Pull request by sea-bass**
This one is actually from my fork...

### CODE REVIEW

**Comment by sea-bass** [comment]
Ok @ct2034 , the lint action actually did not run from forks but I just changed it so it does.

**Comment by sea-bass** [comment]
Ah, we need to only push to GHCR if not on a fork, will do that next.

**Comment by ct2034** [comment]
> Ah, we need to only push to GHCR if not on a fork, will do that next.

No I think that should be good. It will be tagged based on the branch and only main pushes as latest. That’s how I set it up anyway

**Comment by ct2034** [comment]
Ah yes. You are right. Sorry

File: .github/workflows/docker.yml
Comment (ct2034) [inline .github/workflows/docker.yml:1]: You are right

Comment (sea-bass) [reply .github/workflows/docker.yml:1]: I had to go through this on PyRoboSim, so the wound is fresh :)

**Comment by ct2034** [comment]
Can you also add the condition to the release step? Sorry I am on my phone in the keynote of Magnus egerstedt ;)

**Review by facontidavide — APPROVED** [review]

**Review by ct2034 — APPROVED** [review]

**Pull request merged**